installer/test/InstallerFunctionalTests/helpers.cpp

```diff
@@ -110,8 +110,19 @@ namespace WindowsAppRuntimeInstallerTests
 
     ProcessorArchitecture GetSystemArchitecture()
     {
```

Comment:
Isn't this the same as the code above? Why the duplication?

Reply:
One is in the test project, the other is part of the installer.